### PR TITLE
LD-155 Correct image and instructions for How to update Lucee

### PR DESCRIPTION
**Pull request by gavinbaumanis**
Updated the image and instructions for **How to update Lucee**

### CODE REVIEW

**Comment by cfmitrah** [comment]
https://luceeserver.atlassian.net/browse/LD-155

**Comment by zspitzer** [comment]
@gavinbaumanis why did you remove the section about update providers?

File: docs/04.guides/03.updating-lucee/01.update/page.md [hunk 1]
```diff
@@ -7,17 +7,11 @@ id: updating-lucee-update
...
 To update an existing Lucee Installation go to "Services/Update" in your Lucee Server Administrator.
 
-If a patch is available, Lucee will display a box detailing the changes made in this and all intermediate patches. Simply click the "execute update" button to patch your current version.
+If a patch is available, Lucee will display a box detailing the changes made in this and all intermediate patches. Simply click the "execute" button to patch your current version.
 
-![update.png](https://bitbucket.org/repo/rX87Rq/images/1091096189-update.png)
+You can choose from Releases, Pre Releases or Snapshots (Bleeding Edge). You can even define a custom update provider for your builds!
 
-## Update Providers ##
-
```
Comment (zspitzer) [inline docs/04.guides/03.updating-lucee/01.update/page.md:15]: why did you remove this?

**Review by zspitzer — CHANGES_REQUESTED** [review]

**Comment by gavinbaumanis** [comment]
Hi @zspitzer 
Just checking in...
AFtewr my explanation...
Do you still want the UPDATE PROVIDERS heading left in?

**Pull request merged**